hopper/setup.py

```diff
@@ -80,6 +80,40 @@
     _maybe_write,
 )
 
+def create_build_config_file():
```

Comment:
I'm confused about this file that's being written out. It looks like the Fake kernels don't use this? (am I reading this correctly?)

Reply:
nvm I see it in round_up_headdim